policyengine_us/parameters/gov/states/hi/tax/income/credits/cdcc/dependent_care_benefits/expense_floor/one_child.yaml

```diff
@@ -0,0 +1,19 @@
+description: Hawaii limits the qualified expenses under the child and dependent care credit for one child/dependent to this amount.
```

Comment:
Could we unify this with the other as a scale parameter?

Reply:
Done!